sdk/nodejs/helm.ts

```diff
@@ -10,17 +10,32 @@ import * as path from "./path";
 import * as nodepath from "path";
 
 export namespace v2 {
-    export interface ChartOpts {
+    interface BaseChartOpts {
+        namespace?: string;
+        values?: any;
+        transformations?: ((o: any) => void)[];
+    }
+
+    export interface ChartOpts extends BaseChartOpts {
         repo: string;
         chart: string;
         version: string;
 
-        namespace?: string;
-        values?: any;
-        transformations?: ((o: any) => void)[];
         fetchOpts?: FetchOpts;
     }
 
+    function isChartOpts(o: any): o is ChartOpts {
+        return "repo" in o && "chart" in o && "version" in o;
+    }
+
+    export interface LocalChartOpts extends BaseChartOpts {
+        directory: string;
```

Comment:
Nit - is `path` or `folder` better here?  Not sure of good precedent to base on - but for some reason `directory` felt a little surprising to me.